ffig/generators/python.py

```diff
@@ -0,0 +1,24 @@
+# Generator module for Python 2 and 3.
+
+import generators
+import os
+
+def generator(module_name, binding, api_classes, env, args, output_dir):
+    module_dir = os.path.join(output_dir, module_name)
```

Comment:
Seems necessary as `args.module_name` and `module_name` may not be the same which would lead to baffling behaviour as different generators would get the info from different places.